tilings/tiling.py

```diff
@@ -1185,6 +1276,13 @@ def requirements(self) -> Tuple[Tuple[GriddedPerm, ...], ...]:
     def total_requirements(self) -> int:
         return len(self._requirements)
 
+    @property
+    def assumptions(self) -> Tuple[AbstractAssumption, ...]:
+        return self._assumptions
+
+    def total_assumptions(self) -> int:
```

Comment:
Maybe `num_assumptions` would be a better name?

Reply:
I used the word total, as other methods used this, e.g. total_obstructions, total_requirements, etc. I can change it if you like

Reply:
Ok I didn't knew it was consistent with the other. Probably not worth changing then